Automattic/jetpack
# CLI: Do not automatically install symlink

Fixes #18564

#### Changes proposed in this Pull Request:
* Removes automatic linking.
* Adds cli function to unlink.
* TODO: Documentation. (Pushing early for confidence check)

#### Jetpack product discussion


n/a

#### Does this pull request change what data or activity we track or use?


n/a

#### Testing instructions:






* yarn install should no longer create a symlink.
* yarn cli-link should create one.
* jetpack cli unlink should remove it.

#### Proposed changelog entry for your changes:


* n/a
